pyproject.toml

```diff
@@ -139,6 +139,7 @@ ani2x = 'molflux.datasets.builders.ani2x.ani2x'
 gdb9 = 'molflux.datasets.builders.gdb9.gdb9'
 pcqm4m_v2 = 'molflux.datasets.builders.pcqm4m_v2.pcqm4m_v2'
 spice = 'molflux.datasets.builders.spice.spice'
+tdc_admet_benchmarks = 'molflux.datasets.builders.tdc.tdc_admet'
```

Comment:
thought: as it needs the `[tdc]` extra to be available, should we  \ do we want to bundle this in the "core" entrypoints or in a separate category (e.g. `[project.entry-points.'molflux.datasets.plugins.tdc']`)